sdk/cognitiveservices/azopenai/constants.go

```diff
@@ -8,6 +8,28 @@
 
 package azopenai
 
+// AzureOpenAIOperationState - The state of a job or item.
+type AzureOpenAIOperationState string
```

Comment:
Since we're not exposing the poller, can these be removed?

Reply:
It's still referenced since we're calling into the poller underneath so I've unexported it.